src/renderer/widgets/EditDelegationModal/model/edit-delegation-model.ts

```diff
@@ -218,6 +218,7 @@ sample({
         accountId: shard.accountId,
         balance: amount || '0',
         conviction: conviction || 'None',
+        previousConviction: activeDelegations[address].conviction || 'None',
```

Comment:
What if there is no active delegation for this address? It will crash

Reply:
For `edit delegation operation` active delegation always exists. If it doesn't exist it will be `delegate operation`